website/contents/pages.js

```diff
@@ -90,6 +90,10 @@ export const examplePages = generatePath([
           code: getCodeUrl('examples/scatterplot')
         }
       },
+      {
+        name: '3D Indoor Scan',
+        external: 'http://uber.github.io/deck.gl/examples/point-cloud-laz'
```

Comment:
If external, move to showcases?

Reply:
This is our base point cloud layer example. It should be integrated when we get around to it.